src/cli/tc.rs

```diff
@@ -31,9 +32,82 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init => init().await,
   }
 }
 
+pub async fn init() -> Result<()> {
+  let tailcallrc = r"directive @server(
+    allowedHeaders: [String]
+    baseURL: String
+    enableApolloTracing: Boolean
+    enableCacheControlHeader: Boolean
+    enableGraphiql: String
+    enableHttpCache: Boolean
+    enableIntrospection: Boolean
+    enableQueryValidation: Boolean
+    enableResponseValidation: Boolean
+    globalResponseTimeout: Int
+    port: Int
+    proxy: Proxy
+    vars: [KeyValue]
+  ) on SCHEMA
+  directive @http(
+    path: String!
+    method: Method = GET
+    query: [KeyValue]
+    body: String
+    baseURL: String
+    headers: [KeyValue]
+  ) on FIELD_DEFINITION
+  directive @inline(path: [String]!) on FIELD_DEFINITION
+  directive @modify(omit: Boolean, name: String) on FIELD_DEFINITION
+  directive @batch(path: [String]!, key: String!) on FIELD_DEFINITION
+  
+  enum Method {
+    GET
+    POST
+    PUT
+    DELETE
+    PATCH
+    HEAD
+    OPTIONS
+  }
+  
+  input Proxy {
+    url: String
+  }
+  
+  input KeyValue {
+    key: String!
+    value: String!
+  }";
+
+  fs::write(".tailcallrc.graphql", tailcallrc)?;
+
+  println!("Do you want to add a file to the project? (yes/no)");
+  let mut input = String::new();
+  io::stdin().read_line(&mut input)?;
+
+  if input.trim() == "yes" {
+    println!("Enter the file name:");
+    let mut file_name = String::new();
+    io::stdin().read_line(&mut file_name)?;
+    file_name = file_name.trim().to_string();
+    fs::write(&file_name, "")?;
+
+    let graphqlrc = format!(
+      r#"schema:
+- "./{}"
+- "./.tailcallrc.graphql"#,
+      file_name
+    );
+    fs::write(".graphqlrc.yml", graphqlrc)?;
+  }
+
+  Ok(())
+}
```

Comment:
Would be great to get some color coding done here. For eg: the questions should be in Yellow. I also think before creating the files, a confirmation should taken from the user. They can say `y/n/q`
- y: create the files
- n: ask the questions again
- q: exit